test/send/sendGCM.js

```diff
@@ -193,6 +194,7 @@ describe('push-notifications-gcm', () => {
           expect(message.params.data.sound).to.be.undefined();
           expect(message.params.data.icon).to.equal('myicon.png');
           expect(message.params.data.msgcnt).to.equal(2);
+          expect(message.params.notification.notification_count).to.eql(42);
```

Comment:
the scope of this test are android specific fields, so we should probably test the handling of `notificationCount` here and have a separate test for the compatibility with `badge`

Reply:
Done